docs/source/quickstart.mdx

```diff
@@ -7,16 +7,16 @@ Each dataset is unique, and depending on the task, some datasets may require add
 <div class="mt-4">
    <div class="w-full flex flex-col space-y-4 md:space-y-0 md:grid md:grid-cols-3 md:gap-y-4 md:gap-x-5">
       <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#nlp"
-         ><div class="w-full text-center bg-gradient-to-br from-emerald-400 to-blue-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">NLP</div>
-         <p class="text-gray-700">Tokenize a dataset and train a model to determine whether a pair of sentences have the same meaning.</p>
+         ><div class="w-full text-center bg-gradient-to-r from-violet-300 via-sky-400 to-green-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Audio</div>
+         <p class="text-gray-700">Resample an audio dataset and get it ready for a model to classify what type of banking issue a speaker is calling about.</p>
       </a>
       <a class="!no-underline border dark:border-gray-700 p-5 rounded-lg shadow hover:shadow-lg" href="/docs/datasets/quickstart#audio"
-         ><div class="w-full text-center bg-gradient-to-br from-violet-400 to-orange-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Audio</div>
-         <p class="text-gray-700">Resample an audio dataset and get it ready for a model to classify what type of banking issue a speaker is calling about.</p>
+         ><div class="w-full text-center bg-gradient-to-r from-rose-300 via-purple-400 to-sky-500 rounded-lg py-1.5 font-semibold mb-5 text-white text-lg leading-relaxed">Vision</div>
```

Comment:
I still don't see the color gradient for vision

Reply:
should be good now https://moon-ci-docs.huggingface.co/docs/datasets/pr_4440/en/quickstart